lib/6to5/transformation/templates/get.js

```diff
@@ -0,0 +1,23 @@
+(function get(object, property, receiver) {
+  var desc = Object.getOwnPropertyDescriptor(object, property);
+
+  if (desc === void 0) {
```

Comment:
Common convention is to use undefined. There's an `undefinedToVoid` optional transformer if this is something that users want.